### DISCUSSION TITLE
I´m not able to install

### DISCUSSION THREAD

**Discussion by SAGARR1998** [jhassan8/crunchyroll-tizen#39] [Q&A] [answered]
After having tried the two methods that you have uploaded I have been unable to make it work, I don't know if the error comes from my use of the Tizen APP, I have managed to connect the PC and the television and it is reflected in the device manager, or with cmd it also says it is connected, but I don't know what else to do...
Would a video tutorial be possible? thanks for the conversion of the app

**Comment by TONYSOAD05** [comment]
Use the "With Installer" option, there is a tutorial there. Following the "only text" tutorials its a little bit hard.

**Comment by SAGARR1998** [comment]
Hi, I tried to install again with the new update (1.1.1), but I got the same issues, I followed every step... Maybe its my tv.
This is the error:
![Screenshot_20230807-120252](https://github.com/jhassan8/crunchyroll-tizen/assets/140554514/dbe9a6f5-7788-41a8-9290-fc5a46e8c8c5)

**Comment by TONYSOAD05** [comment]
I think that error is beacuse you moved the default path for tizen studio, when you install Tizen Studio; you should leave the default path. If not, in the first option you need to put the full path.

**Reply by SAGARR1998** [reply]
I dont know what is failed, anyway,  I reinstall with the default path and I put It manually in the  cmd(install Windows bat) and nothing. THX anyway.
![PXL_20230807_151732704](https://github.com/jhassan8/crunchyroll-tizen/assets/140554514/0585ec07-7455-4ff0-b1d6-859197421f27)

**Reply by TONYSOAD05** [reply, marked as answer]
Oye bro, veo que tienes tu PC en Español, entonces supongo hablas Español. El problema es que estas moviendo algún archivo fuera del Default.

Por ejemplo, en la pantalla de CMD del error dice "Carpeta"; me imagino que moviste los archivos a una carpeta, ahí está el error. En resumen, los pasos son:

-Instalar Tizen Studio en C.
-Ir al TV y activar el Developer Mode y poner ahí la IP del PC. Apagar la TV, esperar un minuto y volverla a prender.
-Descargar el archivo 1.1.1 y descomprimir en la carpeta Descargas. 
-Correr el Bat y poner en el CMD la IP de la TV (yendo al menú Network).
-Y listo, debería de funcionar.

Yo lo hice en 2 TV samsung y sin problemas.

**Comment by SAGARR1998** [comment]
Primero gracias por ser tan atento. En segundo lugar, sí, tenías razón al crear una carpeta donde "descomprimía" los archivos daba error, incluso si dentro de la terminal ponía la ubicación en el archivo. Mil gracias!!!

**Comment by Raysed** [comment]
Hi, im trying to install crunchyroll on my samsung tv. Tested both methods but fail., with the CMD method I get same error message as @SAGARR1998. Installed tizen studio without change any parameters as the guide show me. Set the tv in developer mode and enter my pc:s ip adress and after restart. Open the bat file in CMD as admin, press enter 2 times and last enter my tv:s ip adress and press enter again with the "error: no argument is allowed..." message.
What i doning wrong? :)

**Comment by jhassan8** [comment]
Hi @Raysed just tried the .bat and it still works...

![image](https://github.com/jhassan8/crunchyroll-tizen/assets/55010545/861cbfa4-8edc-4a40-9bbb-ede4bf02b506)

In my case I have the tizen studio installation in **C:/samsung/tizen-studio** that's why I changed it, but if you use the normal installation you don't need to change it, in the second entry I did not write anything, because I have the "crunchyroll_online.wgt" in the same folder, in the IP enter the IP of my TV connected to the same network. and it's just that.

do you have any screenshot of the error?

I'm going to try to make a video of various ways to install it.

regards

**Reply by Raysed** [reply]
Hi, thanks to your replay. Tested your newer version with same results and in several folders in "dowloads" folder. Saw your picture that you had the folder in  "user/documents" and tested it me to, and voila it miracly works and sucefully install.

When I first tried to use tizen stuio I can't import it, no of the many types of zip files I downloaded had it. And when i open it as "open project from system file" that evensay "no platform" and I can't run it or convert it to a tizen project. All pluins who was needed was installed in "package manager". Just a replay if something can be wrong with your folder, hope you understand my bad english ;)
![image](https://github.com/jhassan8/crunchyroll-tizen/assets/59996501/ca332668-27e3-444d-838d-cde0ada6b773)

![image](https://github.com/jhassan8/crunchyroll-tizen/assets/59996501/3b728027-0fb4-4635-86ba-c1efea52cc5a)
